extensions/widgets/treeview/treeview.lcb

```diff
@@ -1948,22 +1969,49 @@ private handler convertArrayToList(in pArray as Array, in pLevel as Integer, in
 	if pArray is empty then
 		return the empty list
 	end if
-	
+
 	variable tKeys as List
 	put the keys of pArray into tKeys
-	
+
+	variable tKeysTemp as List
+	variable tKeysSorted as List
+
+	variable tKey as String
+	variable tKeyLower as String
+
+	if not mSortCaseSensitive then
+		repeat for each key tKey in pArray
+			push the lower of tKey onto back of tKeysTemp
+		end repeat
+	else
+		put tKeys into tKeysTemp
+	end if
+
 	if mSortNumeric then
-		sort tKeys using handler CompareKeysNumeric
+		sort tKeysTemp using handler CompareKeysNumeric
 	else
 		if mSortAscending then
-			sort tKeys in ascending order
+			sort tKeysTemp in ascending order
 		else
-			sort tKeys in descending order
+			sort tKeysTemp in descending order
 		end if
 	end if
+
+	if not mSortCaseSensitive then
+		repeat for each element tKeyLower in tKeysTemp
+			repeat for each element tKey in tKeys
+				if the lower of tKey is tKeyLower then
+					push tKey onto back of tKeysSorted
+				end if
+			end repeat
+		end repeat
+	else
```

Comment:
Is this intended to be used where there may be values that differ only in the case of their characters? (e.g. "Hello" vs "hello") If so, this method won't work correctly as it will replace both values with whatever one appears first in the list